notebooks/official/custom/get_started_vertex_training_xgboost.ipynb

```diff
@@ -0,0 +1,992 @@
+{
```

Comment:
<p>praise: LOVE this. I've noticed that, too often, our notebooks compose the training package for custom jobs without explaining what the structure is.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1665/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>